task/jefe/src/main.rs

```diff
@@ -86,8 +90,14 @@ struct ServerImpl<'s> {
     task_states: &'s mut [TaskStatus; NUM_TASKS],
     deadline: u64,
     reset_reason: ResetReason,
+
+    /// Base address for a linked list of dump areas
     #[cfg(feature = "dump")]
     dump_areas: u32,
+
+    /// Cache of most recently checked dump area
```

Comment:
might be nice to mention what this is used for?

Reply:
Yup, done